File: projectapi/Models/Payment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace projectapi.Models
{
    public class Payment
    {
        public int PaymentId {get;set;}
        public string? Status {get;set;}
        public decimal Amount {get;set;}
        public DateTime PaymentDate {get;set;}
        public string? Mode {get;set;}
        public int UserId {get; set;}
        public int AdmissionId {get; set;}
        public int CourseId {get; set;}
        public string? CreatedBy {get; set;}
        public DateTime? CreatedAt {get;set;}
    }
}

File: angularapp/src/app/Models/Payment.ts
import { Admission } from "./Admission";
import { Course } from "./Course";
 
export interface Payment{
    paymentId: number,
    status:string ,
    amount: number,
    paymentDate:Date,
    mode:string ,
    userId: number,
    admissionId: number,
    courseId: number,
    createdBy?: string,
    createdAt?:Date,
    course:Course,
    admission:Admission
}